product_docs/docs/pem/9/managing_certificates_copy.mdx

```diff
@@ -0,0 +1,436 @@
+
+    ```shell
+    /etc/init.d/postgresql-x restart
+    ```
+
+    On Linux with `systemd`, for example, on a Centos7 host:
+
+    ```shell
+    systemctl restart postgresql-x
+    ```
+
+## Replacing PEM self-signed certificates and keys with trusted CA certificates and keys
+
+Before replacing the PEM self-signed certificates and keys with the trusted third-party CA certificates and keys, you must get those certificates using the steps indicated in the third-party CA. 
+
+After obtaining the trusted third-party CA certificates and keys, perform the following steps: 
+
+- Replace PEM self-signed certificates and keys with trusted CA-signed certificates and keys
+- Install trusted CA root and intermediate certificates
+- Install trusted CA CRL
+- Disable CRL usage by the server if the trusted third-party CA does not provide CRL
+
+### Example: Replacing the server and agent certificates
+
+This example shows you how to replace the self-signed certificates with the trusted CA certificates.
+
+We have PEM 9.1 server on CentOS 7 with EPAS 14 as the backend server and a standalone agent installed on another machine. We have to replace the certificates for the backend server, standalone agent and default agent.
+
+Perform the steps to update the self-signed SSL certificates:
+
+1.  Make sure both the PEM agents and the PEM server are up and running.
+
+1.  Take a backup of the existing SSL keys and certificates. The SSL keys and certificates are stored in the `data` directory under your PEM installation. For example, the default location on a Linux system is:
+
+    `/var/lib/edb-as/x/data`, where `x` is the EDB Postgres Advanced server database version
+
+    Copy the following files, adding an extension to each file to make the name unique:
+
+    -   `ca_certificate.crt`
+    -   `ca_key.key`
+    -   `root.crt`
+    -   `root.crl`
+    -   `server.key`
+    -   `server.crt`
+
+    For example, to create a backup of the `ca_certificate` file with the word `old` appended to the entry, use this command:
+
+    ```shell
+    cp ca_certificate.crt ca_certificate_old.crt
+    ```    
+
+3.  Replace the certificates and keys with the CA certificates and keys
+
+    -   Using intermediate key or root key
+
+    -   Using certificates without key
+
+    **Steps to use intermediate or root key:**
+
+    a. Generate a private key for the server:
+ 
+       ```shell
+       openssl genrsa -des3 -out server.key 4096
+       openssl rsa -in server.key -out server.key
+       ```
+
+    b. Generate a Certificate Signing Request (CSR) for the server:
+
+       ```shell
+       openssl req -new -key server.key -out server.csr -subj '/C=IN/ST=MH/L=Pune/O=PEM/CN=pem.enterprisedb.com'
+       ```
+
+    c. Ask your CA to generate an intermediate key pair and sign the intermediate CSR for your server. After root CA signs the CSR, the intermediate CA can install the `root.crt` along with its own `intermediate.key`. Append your `root.crt` to `intermediate.crt`:
```

Comment:
This step is a little confusing. It might help to break it into two steps: one about getting the intermediate key from the CA (making it clear what things you do and what things the CA does) and one where the user is the actor (Append your root.crt ...)